html5/connect.html

```diff
@@ -22,7 +22,6 @@
     <link rel="icon" type="image/png" href="favicon.png" />
 
     <script type="text/javascript" src="js/RgbHelpers.js"></script>
-    <script type="text/javascript" src="js/ImageDecoder.js"></script>
```

Comment:
Was this meant to be part of this commit?

Reply:
Yes. ImageDecoder is now using createImageBitmap which we can check directly. There is no need for a check-if-we-can-use-ImageDecoder routine anymore. Since the file is called from a worker, we dont need this import anymore.